store/wallet.ts

```diff
@@ -151,7 +167,6 @@ export default class Wallet extends VuexModule {
     if (session) {
       const { accounts, method } = session
       await this.context.dispatch('initWallet', { method, accounts })
-      await this.initWallet({ accounts, method })
```

Comment:
Why remove this line?

Reply:
Since we already have this.context.dispatch('initWallet', { method, accounts }), we probably don't need to repeat this.initWallet?